### PR TITLE
Bringing iOS transceiver init handling closer to Android

### PR DESCRIPTION
**Pull request by 8BallBomBom**
Need to do some testing to ensure everything is working as expected.

### CODE REVIEW

**Comment by 8BallBomBom** [comment]
@saghul Seems to be working as expected 👍🏻 
But might need a fix up.

File: ios/RCTWebRTC/SerializeUtils.m [hunk 1]
```diff
@@ -193,21 +193,65 @@ + (NSString *) serializeDirection: (RTCRtpTransceiverDirection) direction {
...
+      [encoding setScaleResolutionDownBy: (NSNumber *) params[@"scaleResolutionDownBy"]];
+    }
+
+    return encoding;
+}
+
++ (RTCRtpTransceiverInit *) parseTransceiverOptions: (NSDictionary *) params {
+    RTCRtpTransceiverInit *transceiverInit = [RTCRtpTransceiverInit new];
+    
+    NSString *direction = [params objectForKey: @"direction"];
```
Comment (saghul) [inline ios/RCTWebRTC/SerializeUtils.m:236]: Here we should check if the direction is not nil before proceeding, like we do on Android: https://github.com/react-native-webrtc/react-native-webrtc/blob/f2278cf36ddaff43b505f19cad558240a4997aab/android/src/main/java/com/oney/WebRTCModule/SerializeUtils.java#L292

**Review by saghul — CHANGES_REQUESTED** [review]
One nit, LGTM otherwise, good work!

File: ios/RCTWebRTC/SerializeUtils.m (hunk 1 above)
Comment (8BallBomBom) [reply ios/RCTWebRTC/SerializeUtils.m:236]: Just figured as the direction is set as Inactive by default, the same functionality occurs without checking if the direction is nil.

Should i still proceed?

Comment (8BallBomBom) [reply ios/RCTWebRTC/SerializeUtils.m:236]: Ahhh wait alright, yes my thinking of that was backwards, will address that now 👍🏻

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
:rocket:

**Pull request merged**